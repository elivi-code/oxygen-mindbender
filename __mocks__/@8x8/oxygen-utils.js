export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
