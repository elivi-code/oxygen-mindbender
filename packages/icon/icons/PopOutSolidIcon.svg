<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M8 2H20C21.1046 2 22 2.89543 22 4V16C22 17.1046 21.1046 18 20 18H18V20C18 21.1046 17.1046 22 16 22H4C2.89543 22 2 21.1046 2 20V8C2 6.89543 2.89543 6 4 6H6V4C6 2.89543 6.89543 2 8 2ZM6 16C6 17.1046 6.89543 18 8 18H16V20H4V8H6V16ZM12 6C11.4477 6 11 6.44772 11 7C11 7.55228 11.4477 8 12 8H14.5229L10.3059 12.217C9.89803 12.6249 9.89803 13.2862 10.3059 13.6941C10.7138 14.102 11.3751 14.102 11.783 13.6941L16 9.47709V12C16 12.5523 16.4477 13 17 13C17.5523 13 18 12.5523 18 12V7C18 6.44772 17.5523 6 17 6H16.9555H12Z" fill="#636363"/>
</svg>
