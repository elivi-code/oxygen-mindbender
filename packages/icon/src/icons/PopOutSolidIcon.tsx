import React from 'react';
import type { IconProps } from '../utils/createSvgIcon';
import createSvgIcon from '../utils/createSvgIcon';

export type { IconProps };
export default createSvgIcon(<g><path fillRule="evenodd" d="M8 2h12c1.1046 0 2 .8954 2 2v12c0 1.1046-.8954 2-2 2h-2v2c0 1.1046-.8954 2-2 2H4c-1.1046 0-2-.8954-2-2V8c0-1.1046.8954-2 2-2h2V4c0-1.1046.8954-2 2-2ZM6 16c0 1.1046.8954 2 2 2h8v2H4V8h2v8Zm6-10c-.5523 0-1 .4477-1 1s.4477 1 1 1h2.5229l-4.217 4.217c-.4079.4079-.4079 1.0692 0 1.4771.4079.4079 1.0692.4079 1.4771 0L16 9.4771V12c0 .5523.4477 1 1 1s1-.4477 1-1V7c0-.5523-.4477-1-1-1h-5Z" clipRule="evenodd"/></g>, 'PopOutSolidIcon');
