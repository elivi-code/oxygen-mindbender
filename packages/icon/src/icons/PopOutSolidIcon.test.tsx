/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import PopOutSolidIcon from './PopOutSolidIcon';

describe('<PopOutSolidIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<PopOutSolidIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
