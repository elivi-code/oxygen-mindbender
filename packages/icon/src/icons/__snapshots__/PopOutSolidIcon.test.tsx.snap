// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopOutSolidIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      clipRule="evenodd"
      d="M8 2h12c1.1046 0 2 .8954 2 2v12c0 1.1046-.8954 2-2 2h-2v2c0 1.1046-.8954 2-2 2H4c-1.1046 0-2-.8954-2-2V8c0-1.1046.8954-2 2-2h2V4c0-1.1046.8954-2 2-2ZM6 16c0 1.1046.8954 2 2 2h8v2H4V8h2v8Zm6-10c-.5523 0-1 .4477-1 1s.4477 1 1 1h2.5229l-4.217 4.217c-.4079.4079-.4079 1.0692 0 1.4771.4079.4079 1.0692.4079 1.4771 0L16 9.4771V12c0 .5523.4477 1 1 1s1-.4477 1-1V7c0-.5523-.4477-1-1-1h-5Z"
      fillRule="evenodd"
    />
  </g>
</svg>
`;
